resources/views/pages/register.twig

```diff
@@ -0,0 +1,133 @@
+                            </div>
+                        {% endif %}
+                    </div>
+                    {% endif %}
+                    {% if not (config('enable_pending_registrations') and is_user()) %}
+                        <div class="row">
+                            <div class="col-sm-6">
+                                {{ f.input('password', __('Password'), 'password', true) }}
+                            </div>
+                            <div class="col-sm-6">
+                                {{ f.input('password_confirmation', __('Confirm password'), 'password', true) }}
+                            </div>
+                        </div>
+                    {% endif %}
+                    <div class="form-group">
+                        <label>
+                            {{ __('What do you want to do?') }}
+                            (<a href="">{{ __('Description of job types') }}</a>)
+                        </label>
+                    </div>
+                    {% for type in angel_types() %}
+                        {{ f.checkbox('angel_types_' ~ type.id, type.name ~ (type.restricted ? ' (restricted)'), not type.restricted) }}
+                    {% endfor %}
+                    <span class="help-block">
+                        {{ m.glyphicon('info-sign') }}
+                        {{ __('Restricted angel types need will be confirmed later by a supporter. You can change your selection in the options section.') }}
+                    </span>
+                </div>
+                <div class="col-md-6">
+                    <div class="row">
+                        {% if config('enable_dect') %}
+                            <div class="col-sm-6">
+                                {{ f.input('dect', __('DECT'), 'text') }}
+                            </div>
+                        {% endif %}
+                        <div class="{{ config('enable_dect') ? 'col-sm-6' : 'col-sm-12' }}">
+                            {{ f.input('mobile', __('Mobile'), 'text') }}
+                        </div>
+                    </div>
+                    {% if config('enable_user_name') %}
+                        <div class="row">
+                            <div class="col-sm-6">
+                                {{ f.input('firstname', __('First name'), 'text') }}
+                            </div>
+                            <div class="col-sm-6">
+                                {{ f.input('lastname', __('Last name'), 'text') }}
+                            </div>
+                        </div>
+                    {% endif %}
+                    <h4>
+                        {{ f.entry_required() }}
+                        = {{ __('Entry required!') }}
+                    </h4>
+                </div>
+            </div>
+            <div class="form-group">
+                <button class="btn btn-primary" type="submit" name="submit">
+                    {{ __('Register') }}
```

Comment:
As above: This should be `form.register` or (as it's only used here) `register.register`